src/modules/job-manager/restart.c

```diff
@@ -241,7 +241,7 @@ int restart_save_state (struct job_manager *ctx)
 
     if (!(txn = flux_kvs_txn_create ())
         || restart_save_state_to_txn (ctx, txn) < 0
-        || !(f = flux_kvs_commit (ctx->h, NULL, 0, txn))
+        || !(f = flux_kvs_commit (ctx->h, NULL, FLUX_KVS_SYNC, txn))
```

Comment:
Again - high cost operation, and we already have `kvs.checkpoint-period` periodically saving the KVS root.